libs/output-mapping/src/Writer/TableWriter.php

```diff
@@ -259,21 +263,29 @@ private function createLoadTableTask(
         return $loadTask;
     }
 
-    private function ensureValidDestinationBucketExists(MappingDestination $destination, array $systemMetadata)
+    private function ensureDestinationBucket(MappingDestination $destination, array $systemMetadata): array
```

Comment:
imho should be typehinted, the random arrays from sapi client are hell

Reply:
maybe it would be better to return just [id => $id ,backend => $backend] because that's all we need

Reply:
ok